backend/src/main/resources/logging/logback-spring.xml

```diff
@@ -0,0 +1,51 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<configuration>
+    <property name="LOG_HOME" value="./src/main/resources/logging/logs"/>
```

Comment:
Aktuell ist das so in Ordnung aber auf dauer glaube ich müssen wir an einen anderen Ort schreiben stimm dich bitte mit Luca ab, wie ihr das handhaben wollt

Reply:
Alles klar

Reply:
Luca meinte, dass das da durchaus erstmal verbleiben soll

Reply:
Markiere das dann jetzt als resolved